Automattic/jetpack
# Update: license activation dialog text

#### Changes proposed in this Pull Request:

This PR is to fix inconsistencies in the wording of Activation Dialog, which is shown after activating the Jetpack license manually. Mostly, we just need to remove 'Yours' from the title.
The full description can be found here: 1164141197617539-as-1201710570188029/f

#### Does this pull request change what data or activity we track or use?
This PR only changes wording, so I assume it doesn't changes anything for that matter.

#### Testing instructions:
- Build the brunch locally (`js-packages/licensing` and `plugins/jetpack`)
- Up docker locally and create a tunnel to expose your container.
- For each of the products (except `Search`) on [cloud.jetpack](https://cloud.jetpack.com/pricing) do the following steps
- - Remove/cancel all your plans
- - Purchase a new plan at [cloud.jetpack](https://cloud.jetpack.com/pricing) and copy license key
- - In your wp-admin->Jetpack activate the plan using a license key and observe the Dialog widow shown after.

***

#### Expected result
The expected result is per screenshots attached below:

<img width="1126" alt="Screenshot 2022-01-31 at 14 31 37" src="https://user-images.githubusercontent.com/60262784/151856045-5297d493-29d0-4b92-8a3b-d0d59f0c1f7c.png">

<img width="623" alt="Screenshot 2022-01-31 at 15 10 41" src="https://user-images.githubusercontent.com/60262784/151856328-410947d6-6d3c-4e84-882b-6564e1ad3b9e.png">

<img width="658" alt="Screenshot 2022-01-31 at 15 08 55" src="https://user-images.githubusercontent.com/60262784/151856333-46a57980-7910-419f-bb37-2dea5dccc1d0.png">

<img width="651" alt="Screenshot 2022-01-31 at 15 02 22" src="https://user-images.githubusercontent.com/60262784/151856356-0859aa67-188e-46ef-9eff-4753174b3cdb.png">

<img width="527" alt="Screenshot 2022-02-02 at 13 24 47" src="https://user-images.githubusercontent.com/60262784/152145039-6dfea468-711c-477d-8410-d9d4b3a970fb.png">

<img width="438" alt="Screenshot 2022-01-31 at 14 52 06" src="https://user-images.githubusercontent.com/60262784/151856394-dd4ca2f4-f5d8-4615-b58b-6d946c2cc090.png">
